packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -192,16 +230,10 @@ const DatePicker = React.memo(props => {
                 <Native
                     id={id}
                     name={name}
-                    display_format={display_format}
-                    error={error}
-                    label={label}
-                    max_date={max_date}
-                    min_date={min_date}
                     onBlur={onBlur}
                     onFocus={onFocus}
                     onSelect={onSelectCalendarNative}
-                    placeholder={placeholder}
-                    value={getCalendarValue(date)} // native picker accepts date format yyyy-mm-dd
+                    value={getCalendarValue(date as string) as string} // native picker accepts date format yyyy-mm-dd
```

Comment:
```suggestion
                    value={getCalendarValue(date)} // native picker accepts date format yyyy-mm-dd
```
we should not assert string type here because both Native and Calendar components assume that value that we pass into them can be a non-string value and have code to handle the non-string value inside them. So it's fine to pass `getCalendarValue(date)` as is. If you can access types of Native and Calendar components in your branch, please update their props type to accept null value besides a string value.👍🏻